### ISSUE TITLE
Editor buffers the texts input

### ISSUE THREAD

**Issue by ericleethrinacia** [froala/angular-froala#55]
I've already addressed this in email but just want to leave it here for a record

The editor seems to buffer when the user types. By buffer I mean if one types just a bit faster, the editor slowly shows the characters that are just typed even you have done typing
I commented out keyup and contentChanged event in _init function at line 7191 to prevent this from happening but I was told that it wouldn't be good practice and I have noticed by doing so, the undo button didn't refresh itself after contentChanged. I was also told that it could be due to angular binding.

Thank you very much

**Comment by stefanneculai** [comment]
@cgallarno I believe this is related to https://github.com/froala/angular-froala/blob/master/src/angular-froala.js#L53. Any thoughts?

**Comment by ericleethrinacia** [comment]
@stefanneculai Sorry, I forgot to mention, I currently commented out that line as well since I found it was in a way duplicating what the contentChanged event just below is doing. But the speed is still not great so I commented out the bulkrefresh in keyup and contentChanged event.

Also, advise me if I need to start new issue for this and where. The editor seems to save last step every 500 ms. It makes sense to do that but for some reason every time it's called, the favicon and mouse indicator flashes
